spec/unit/accordion/template.js

```diff
@@ -0,0 +1,9 @@
+var template = [
```

Comment:
I would love for this to be an actual HTML or compiled-to-HTML file, although I'm not clear exactly how the consuming client would be able to drop that in to their application without have a build process themselves. I guess that's the questions everyone is asking though :D